packages/react-components/react-tree/src/components/TreeItem/useTreeItemStyles.styles.ts

```diff
@@ -22,14 +22,9 @@ const useRootStyles = makeStyles({
   base: {
     position: 'relative',
     cursor: 'pointer',
-    display: 'grid',
+    display: 'flex',
```

Comment:
`TreeItem` becomes a column oriented flexbox. As all possible visual items are now part of the layout component the `TreeItem` itself ends up having only 2 possible children:

1. Layout component
2. Subtree